### ISSUE TITLE
Sentry Relay: Network Outage

### ISSUE THREAD

**Issue by mpass99** [openHPI/poseidon#364]
Both the staging and the production Sentry relay repeat the warning in their Docker logs (`sudo docker logs -f relay`):
```
2023-05-05T23:00:26Z [relay_server::actors::upstream] WARN: Network outage, scheduling another check in 60s
```

**Comment by MrSerth** [comment]
I identified the root cause for this issue: The (unavailability of the) DNS servers provided. In our current setup, we have multiple DNS servers, one on the public interface and others on the internal one. For a yet unknown reason, the public-facing DNS servers are preferred but cannot be reached. For some name resolution, this "just" causes a delay, whereas the Relay obvious fails. 

I deployed a Docker-specific solution (valid until the next deployment; keeping the current global configuration for debugging) and will continue investigating.
